src/main/java/io/Out.java

```diff
@@ -0,0 +1,153 @@
+package io;
+
+import tasks.Store;
+import tasks.Task;
+
+public class Out {
+    private static final int BOX_WIDTH = 80;
+
+    private static void printLineWithDelim(char leftDelim, char rightDelim) {
+        System.out.print(leftDelim);
+        for (int i = 0; i < BOX_WIDTH - 2; i++) {
+            System.out.print(BoxDrawingCharacter.HORIZONTAL_LINE);
+        }
+        System.out.println(rightDelim);
+    }
+
+    private static void printBoxTopBorder() {
+        printLineWithDelim(BoxDrawingCharacter.TOP_LEFT_CORNER, BoxDrawingCharacter.TOP_RIGHT_CORNER);
+    }
+
+    private static void printBoxBottomBorder() {
+        printLineWithDelim(BoxDrawingCharacter.BOTTOM_LEFT_CORNER, BoxDrawingCharacter.BOTTOM_RIGHT_CORNER);
+        System.out.println("");
+    }
+
+    private static void printBoxBottomBorder(boolean withBottomMargin) {
+        printLineWithDelim(BoxDrawingCharacter.BOTTOM_LEFT_CORNER, BoxDrawingCharacter.BOTTOM_RIGHT_CORNER);
+        if (withBottomMargin) {
+            System.out.println("");
+        }
+    }
```

Comment:
Is printBoxBottomBorder() necessary? It has the same function as printBoxBottomBorder(false).

Reply:
Yes, the presence of both is intentional, I used overloading here to avoid having to use `printBoxBottomBorder(false)` all the time, as [Java does not allow optional parameters](https://stackoverflow.com/a/12994104).